aspnetcore/fundamentals/servers/aspnet-core-module.md

```diff
@@ -0,0 +1,85 @@
+---
+title: ASP.NET Core Module overview
+author: tdykstra
+ms.author: tdykstra
+manager: wpickett
+ms.date: 10/27/2016
+ms.topic: article
+ms.assetid: 4661af33-34c5-4d71-93a0-8c7632f43580
+ms.prod: aspnet-core
+uid: fundamentals/servers/aspnet-core-module
+---
+# ASP.NET Core Module overview
+
+By [Tom Dykstra](http://github.com/tdykstra), [Rick Strahl](https://github.com/RickStrahl), and [Chris Ross](https://github.com/Tratcher) 
+
+ASP.NET Core Module (ANCM) is an IIS module that lets [Kestrel](kestrel.md) use IIS or IIS Express as a reverse proxy server. **ANCM works only with Kestrel; it isn't compatible with [WebListener](weblistener.md).** 
+
+Supported Windows versions:
+
+* Windows 7 and Windows Server 2008 R2 and later
+
+[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/servers/aspnet-core-module/sample)
+
+## What ASP.NET Core Module does
+
+ANCM is a native IIS module that hooks into the IIS pipeline very early in the request cycle and immediately redirects all traffic to the backend ASP.NET Core application. All requests - even those mapped to top level Handlers like ASPX - bypass the IIS pipeline and are forwarded to the ASP.NET Core application.
+
+Because ASP.NET Core applications run in a process separate from the IIS worker process, ANCM also does process management. ANCM starts the process for the ASP.NET Core application when the first request comes in and restarts it when it crashes. This is essentially the same behavior as classic ASP.NET applications that run in-process in IIS and are managed by WAS (Windows Activation Service).
+
+Here's a diagram that illustrates the relationship between IIS, ANCM, and ASP.NET Core applications.
+
+![ASP.NET Core Module](aspnet-core-module/_static/ancm.png)
```

Comment:
This image does not explain why there are two instances of IIS and three instances of Kestrel.